doc/README.rst

```diff
@@ -0,0 +1,37 @@
+Chapel Documentation
+====================
+
+This directory contains Chapel's documentation in two formats:
+
+``html/`` contains a web-rendered format of the documentation, which reflects
+the online documentation hosted at ``http://chapel.cray.com/docs``. This can be
+viewed by pointing a web browser of your choice to ``html/index.html``.
+
+``rst/`` contains the documentation in the format of reStructuredText (rst), a
+markup language. This documentation can be viewed with a text editor of your
+choice.
+
+Note that some files in ``rst/`` are written or generated for the purpose of
+being rendered as HTML, such as the ``modules/`` and ``builtins/`` directory,
+as well as the ``index.rst`` files. For this reason, we recommend browsing the
+documentation in the web format.
+
+Resources
+---------
+
+Here are some useful resources available in ``{rst,html}/``:
+
+====================================  ========================================
+Building from source                  ``usingchapel/QUICKSTART.{rst,html}``
+Platform-specific notes               ``platforms/``
```

Comment:
Not sure either. My rationale was that since this file is intended for raw-text consumption, and so browsing the directory will be more helpful than the index.rst, which is mostly sphinx-syntax:

```rst
Platform-Specific Notes
=======================

Major Platforms
---------------

.. toctree::
   :maxdepth: 1

   macosx
   cray
   knl

...
```